frontend/src/scenes/max/TicketPrompt.tsx

```diff
@@ -0,0 +1,179 @@
+ * - If no `summary`: shows input field for user to describe their issue
+ */
+export function TicketPrompt({ conversationId, traceId, summary }: TicketPromptProps): JSX.Element {
+    const [issueText, setIssueText] = useState('')
+    const [isSubmitting, setIsSubmitting] = useState(false)
+    const [isSupportModalOpen, setIsSupportModalOpen] = useState(false)
+
+    const { sendSupportRequest, lastSubmittedTicketId } = useValues(supportLogic)
+    const { resetSendSupportRequest, setSendSupportRequestValue, submitSendSupportRequest, closeSupportForm } =
+        useActions(supportLogic)
+    const { addMessage } = useActions(maxThreadLogic)
+
+    const [pendingTicketSubmission, setPendingTicketSubmission] = useState(false)
+    const [ticketIdBeforeSubmission, setTicketIdBeforeSubmission] = useState<string | null>(null)
+
+    const messageContent = summary || issueText
+
+    const handleTicketCreated = useCallback(
+        async (ticketId: string): Promise<void> => {
+            posthog.capture('posthog_ai_support_ticket_created', {
+                $ai_conversation_id: conversationId,
+                $ai_session_id: conversationId,
+                $ai_trace_id: traceId,
+                $ai_support_ticket_id: ticketId,
+            })
+
+            // Persist the confirmation message to the conversation
+            const confirmationMessage = formatConfirmationMessage(ticketId)
+            await api.conversations.appendMessage(conversationId, confirmationMessage)
+
+            // Add the message to the local thread state
+            addMessage({
+                type: AssistantMessageType.Assistant,
+                content: confirmationMessage,
+                id: uuidv4(),
+                status: 'completed',
+            })
+
+            setIsSupportModalOpen(false)
+            setIsSubmitting(false)
+            closeSupportForm()
+        },
+        [conversationId, traceId, addMessage, closeSupportForm]
+    )
+
+    useEffect(() => {
+        if (pendingTicketSubmission && lastSubmittedTicketId && lastSubmittedTicketId !== ticketIdBeforeSubmission) {
+            void handleTicketCreated(lastSubmittedTicketId)
+            setPendingTicketSubmission(false)
+        }
+    }, [lastSubmittedTicketId, pendingTicketSubmission, ticketIdBeforeSubmission, handleTicketCreated])
+
+    function appendMetadataToMessage(message: string): string {
+        const metadataLines = [`Conversation ID: ${conversationId}`, traceId ? `Trace ID: ${traceId}` : null].filter(
+            Boolean
+        )
+        return message ? `${message}\n\n----\n${metadataLines.join('\n')}` : metadataLines.join('\n')
+    }
+
+    function openSupportModal(): void {
+        resetSendSupportRequest({
+            name: '',
+            email: '',
+            kind: 'bug',
+            target_area: 'max-ai',
+            severity_level: 'low',
+            message: messageContent,
+        })
```

Comment:
I love this flow, thanks!

One small thing from when I worked on it previously, I'd added a [tag to the ticket](https://github.com/PostHog/posthog/pull/40169/files#diff-9aba70a7dcdee8c1a2af6e71600169cf5e528d5058ce828f071fe1da86ca867dR46) to flag that it had come from a PostHog AI conversation - this will make it a bit easier for us to identify/track these requests in Zendesk and potentially build some extra tooling. I still think this would be helpful, but also happy to add that myself later